### ISSUE TITLE
Bib file does not include volume titles

### ISSUE THREAD

**Issue by retorquere** [retorquere/zotero-better-bibtex#2228]
### Discussed in https://github.com/retorquere/zotero-better-bibtex/discussions/2227

<div type='discussions-op-text'>

<sup>Originally posted by **moiken-hinrichs** July 28, 2022</sup>
Hello,
I'm working with RMarkdown/Bookdown and Zotero as my library, which generally is working quite nice. I just noted, that all the volume titles are missing in the bibliography after rendering the document. First I thought it's the .csl which is at fault (using taylor-and-francis-harvard-v) but when I checked the .bib file, I saw that the titles were not included.

An exampe:
```
@inbook{pelegrin2000,
	title = {Les techniques de débitage laminaire au Tardiglaciaire: critères de diagnose et quelques réflexions},
	author = {Pelegrin, Jacques},
	editor = {Valentin, Boris and Bodu, Pierre and Christensen, Marianne},
	year = {2000},
	date = {2000},
	pages = {73{\textendash}86},
	series = {Mémoires du Musée de Préhistoire d'lle de France},
	address = {Nemours}
}
```
Thats how the file looks for all the @inbook, clearly missing the title after the editior. Is there a way I can change that, so I don't have to manualy add the titles after finishing the writing process?</div>

Log ID: P32R5Z9G-refs-euc

**Comment by retorquere** [comment]
@moiken-hinrichs please post a BBT support log ID (which must have `-refs` in it) here.

**Comment by retorquere** [comment]
@moiken-hinrichs with what translator did you export the entry you posted? I see both `year` and `date` and that's not something that BBT does/should be doing. I can't reproduce it in any case.

**Comment by moiken-hinrichs** [comment]
I copy-pasted from the .bib file, which I opened with the windows editor. If thats what you wanted to know. The rendered citation in the bibliography looks like:
Pelegrin, J., 2000. Les techniques de débitage laminaire au tardiglaciaire: Critères de diagnose et quelques réflexions. In: B. Valentin, P. Bodu, and M. Christensen, eds. Nemours, 73–86.

**Comment by retorquere** [comment]
But how did you create the bib file? None of the following exports what was posted above from `P32R5Z9G-refs-euc`:

* Better BibTeX
* Better BibLaTeX
* BibTeX
* BibLaTeX

**Comment by moiken-hinrichs** [comment]
Frankly, I have no idea. I didn't change anything in the YAML settings, so it should be BibTeX.

**Comment by retorquere** [comment]
I don't think this is anything that Zotero/BBT is doing. It is more likely that something in the RMarkdown pipeline (which means likely pandoc) is creating this bib file. I can't think of a way to generate this bib from Zotero/BBT.

**Comment by retorquere** [comment]
How does RStudio (I take it?) interact with Zotero/BBT? I think the culprit will be there.

**Comment by moiken-hinrichs** [comment]
If I remember correctly, I used this instruction: [https://gsverhoeven.github.io/post/zotero-rmarkdown-csl/](url)
And seems I went for the citr version. This package is at least installed in my RStudio. But I'm not sure, as I use the simple shortcut of the visual mode and not the addin via citr.
And here: [https://rstudio.github.io/visual-markdown-editing/citations.html](url) it is stated, that no further configuration is needed, when RStudio and Zotero are run on the desktop. Which I do. 
I have no idea. Maybe I should try to export my library as bib file and use that instead of letting Rmarkdown compile a list.
But thanks for the help. :)
